cisco_sdwan/assets/monitors/device_reboot.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": 2,
+  "created_at": "2024-04-05",
+  "last_updated_at": "2024-04-05",
+  "title": "Cisco SD-WAN Device has rebooted several times",
+  "tags": [
+    "integration:cisco-sdwan"
+  ],
+  "description": "This monitor detects instances of repeated reboots, serving as an indicator for potential issues like software-related glitches, low memory conditions, corrupt software images, hardware faults, or software bugs. By identifying these concerns, it ensures network stability and optimized performance.",
+  "definition": {
+	"name": "[Cisco SD-WAN] Device {{hostname.name}} ({{system_ip.name}}) rebooted more than 5 times in the last 10 minutes",
+	"type": "query alert",
+	"query": "sum(last_10m):sum:cisco_sdwan.reboot.count{*} by {device_namespace,hostname,system_ip} > 5",
```

Comment:
5 reboots instinctively seems quite a lot to me. I'm wondering if we should have a lower threshold, like 3 for instance?
Do you have a particular reason to use 5

Reply:
Good question, no particular reason except I wanted to avoid this monitor to trigger on every maintenance operation (where I imagine you could reboot your device several times in a 10 mins window). But 3 should be fine

Reply:
Updated in 971572b